devtools-ci-index.yaml

```diff
@@ -725,6 +737,17 @@
     <<: *job_template_build_defaults
 
 - job-template:
+    name: '{ci_project}-{git_repo}-build-che-credentials-pr-check'
+    wrappers:
+        - che_functional_tests_credentials_wrapper
+        - che_credentials_wrapper
+        - registry_devshift_credentials
```

Comment:
You don'T need this if it's just a pr-chek - assuming you don't need to push to registry (if you do, then you need it:))